tests/cases/compiler/jsxNamespaceImplicitImportJSXNamespaceFromConfigPickedOverGlobalOne.tsx

```diff
@@ -1,5 +1,5 @@
 // @strict: true
-// @jsx: react-jsx
+// @jsx: preserve,react-jsx
```

Comment:
I have no idea what's expected result of `@jsx: react` + `@jsxImportSource: @emotion/react` though. It seems like it could be considered an invalid combination?

Reply:
We could issue an error in `program.ts` (within `verifyCompilerOptions`) - that's probably fine.